drivers/metal/rendering_device_driver_metal.mm

```diff
@@ -3624,6 +3633,53 @@ bool isArrayTexture(MTLTextureType p_type) {
 	return PipelineID(pipeline);
 }
 
+#pragma mark - Raytracing
+
+// ----- ACCELERATION STRUCTURE -----
+
+RDD::AccelerationStructureID RenderingDeviceDriverMetal::blas_create(BufferID p_vertex_buffer, uint64_t p_vertex_offset, VertexFormatID p_vertex_format, uint32_t p_vertex_count, BufferID p_index_buffer, IndexBufferFormat p_index_format, uint64_t p_index_offset_bytes, uint32_t p_index_count, RDD::BufferID p_transform_buffer, uint64_t p_transform_offset) {
+	// TODO
```

Comment:
`CRASH_NOW_MSG("Unimplemented!");` for these TODOs?

Reply:
I'd use `ERR_FAIL_MSG()` instead to avoid crashing the engine.